src/modules/Dashboard/features/RespondentData/RespondentDataReview/EmptyResponses/EmptyResponses.tsx

```diff
@@ -0,0 +1,23 @@
+import { StyledEmptyReview, StyledWrapper } from './EmptyResponses.styles';
+import { EmptyResponsesProps } from './EmptyResponses.types';
+import { renderEmptyState } from './EmptyResponses.utils';
+
+export const EmptyResponses = ({
+  hasAnswers,
+  isActivityOrFlowSelected,
+  isAnswerSelected,
+  error,
+  'data-testid': dataTestid,
+}: EmptyResponsesProps) => {
+  if (hasAnswers && isActivityOrFlowSelected && isAnswerSelected && !error) {
+    return null;
+  }
+
+  return (
+    <StyledWrapper data-testid={dataTestid}>
+      <StyledEmptyReview>
+        {renderEmptyState({ isAnswerSelected, isActivityOrFlowSelected, error })}
```

Comment:
Could you please consider (in next PR) to avoid the function return in favor of react component?